### ISSUE TITLE
"Custom Plugin" field in "Editor - TinyMCE" settings has no effect

### ISSUE THREAD

**Issue by white961** [joomla/joomla-cms#20236]
### Steps to reproduce the issue

Set the name of a tinymce plugin in the "Custom Plugin" field of the "Editor - TinyMCE" configuration page of the admin ui. 

### Expected result

When an article is edited the specified tinymce extension should be loaded.

### Actual result

When an article is edited the specified tinymce extension is not loaded.

### System information (as much as possible)

In the PlgEditorTinymce class, the onDisplay function calls $levelParams->get('custom_plugin', '') to retrieve the names of custom plugins that the user wants tinymce to load.  The value returned is empty.  So, the extensions the user wants loaded, are not loaded.

Basically, the text the user typed into the plugin configuration is not making it to the code that creates the javascript to initialize tinymce.

### Additional comments

A simple work around is to insert something like this

		$plugins[] = 'nameofplugin';

in the onDisplay function of the PlgEditorTinymce class.

A good plugin to use for testing is the "noneditable" plugin since it's already included with Joomla. 

And of course, this will break when tinymce.php gets overwritten in a future update.  I hope it gets fixed before then.

**Comment by brianteeman** [comment]
i can not confirm this - please make sure that you are adding the plugin to the correct group

i tested by adding a fictional plugin called brian and got an error message in the editor that the plugin didnt exist so tinymce must have tried to load it

**Comment by white961** [comment]
I removed the change to the PlgEditorTinymce class, added the plugin to all of the sets in the tinymce configuration and it works.  

You can close this issue.

Sorry, for wasting your time.
